ansible/recreateDesignDocs.yml

```diff
@@ -0,0 +1,7 @@
+---
+# This playbook wipes the database. Pls be sure you know what you are doing now!
```

Comment:
how about a more ominous comment: `WARNING: This playbook wipes the database. This action is not reversible. Be very careful and know what you are doing.`